app/views/requests/index.html.erb

```diff
@@ -22,8 +23,8 @@
                   <span><%= request.receiver.username %></span>
                 <% end %>
                 <strong><%= request.item.name %></strong>
-                <% if request.item.status == 2 %>
-                  <em><%= '<This item was donated>' if request.item.status == 2 %></em>
+                <% if request.item.status == 'donated' %>
```

Comment:
You can use `request.item.donated?` thanks to enum.

Reply:
yeah smart use of enums! I will change in my future branches XD